packages/scene/src/Values/Internal/Vec3.ts

```diff
@@ -1,9 +1,11 @@
-import { parseSafeFloats, toSafeString } from '../../../../parseFloats';
 import {
-  clamp,
   EPSILON,
-  equalsTolerance
-} from '../../../Core/Values/Internal/Common';
+  equalsTolerance,
+  parseSafeFloats,
+  toSafeString
+} from '@behave-graph/core';
+import { clamp } from 'three/src/math/MathUtils';
```

Comment:
We should consider using [three-stdlib](https://github.com/pmndrs/three-stdlib) - it exports the necessary three.js functionality, but is built in a modern way that supports tree-shaking and ultimately a smaller build size.